### ISSUE TITLE
Unable to compile on Mac OSX 10.11

### ISSUE THREAD

**Issue by piyushhtree** [hyrise/sql-parser#14]
Running make command gives following error.

<img width="1154" alt="screen shot 2016-03-17 at 8 06 10 pm" src="https://cloud.githubusercontent.com/assets/16499638/13849338/bee8555e-ec7b-11e5-92ba-4a103bcfeb1c.png">

**Comment by iammapping** [comment]
Modify the Makefile to solve this issue, mention at #7 by *rressi*.

> `uint` is not standard and probably deprecated also under gcc + Linux, `unsigned` can be used instead.

change
```
CFLAGS     = -std=c++11 -Wall -fPIC
```
to
```
CFLAGS     = -std=c++11 -Wall -fPIC -Duint=unsigned
```
and
```
CTESTFLAGS = -Wall -Isrc/ -Itest/ -L./ -std=c++11 -lstdc++
```
to
```
CTESTFLAGS = -Wall -Isrc/ -Itest/ -L./ -std=c++11 -lstdc++ -Duint=unsigned
```

**Comment by torpedro** [comment]
This should be fixed by the PR #17 which was just merged. Please confirm.
